contrib/kubernetes/skydive.yaml

```diff
@@ -105,6 +105,8 @@ spec:
         env:
         - name: SKYDIVE_ANALYZERS
           value: "$(SKYDIVE_ANALYZER_SERVICE_HOST):$(SKYDIVE_ANALYZER_SERVICE_PORT_API)"
+        - name: SKYDIVE_RUNC_PATH
```

Comment:
it should be SKYDIVE_AGENT_TOPOLOGY_RUNC_RUN_PATH. It seems there is an issue in the openshift template. Can you please fix it is this PR too ?

Reply:
ok - fixed